responses/lv/HassRespond.yaml

```diff
@@ -0,0 +1,10 @@
+language: lv
+responses:
+  intents:
+    HassRespond:
+      default: ""
+      hello: "Sveicieni no Home Assistant."
+      listening: "Nē, Es klausos tikai tad, kad tu pasaki modināšanas frāzi."
```

Comment:
```suggestion
      listening: "Nē, Es klausos tikai tad, kad tu mani uzrunā."
```
Maybe like this?

Reply:
I would like somehow to point out wakeup phrase. Because it does not listen on any "mani uzrunā"

Reply:
Yes, that sounds good as well 👍🏻